community/consistency-check/src/main/java/org/neo4j/consistency/ConsistencyCheckSettings.java

```diff
@@ -56,5 +57,6 @@
     @SuppressWarnings("unchecked")
     @Description("File name for inconsistencies log file. If not specified, logs to a file in the store directory.")
     public static final
-    Setting<File> consistency_check_report_file = setting( "tools.consistency_checker.report_file", PATH, NO_DEFAULT );
+    Setting<File> consistency_check_report_directory = setting( "tools.consistency_checker.report_directory", PATH,
+            GraphDatabaseSettings.logs_directory.getDefaultValue() );
```

Comment:
I think this setting should go altogether and just use `dbms.directories.logs` instead.